<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>578</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>643</width>
    <height>577</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>643</width>
    <height>585</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget_1" native="true">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>6</y>
      <width>512</width>
      <height>530</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="text">
        <string>手写输入0-9：</string>
       </property>
       <property name="textFormat">
        <enum>Qt::PlainText</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MouseTrackerWindow" name="mouseTrackerWindow" native="true">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>500</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>522</x>
      <y>6</y>
      <width>113</width>
      <height>428</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="LineWidthEdit" name="lineWidthEdit" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>133</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>133</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>10</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>90</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>90</height>
        </size>
       </property>
       <property name="title">
        <string>工具选择</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QRadioButton" name="pen">
          <property name="text">
           <string>画笔</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="eraser">
          <property name="text">
           <string>橡皮擦</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="clearButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>101</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>清空输入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="predictButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>101</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>开始预测</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="lcdnumber">
       <property name="minimumSize">
        <size>
         <width>101</width>
         <height>101</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>101</height>
        </size>
       </property>
       <property name="font">
        <font>
         <underline>false</underline>
        </font>
       </property>
       <property name="mouseTracking">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="smallDecimalPoint">
        <bool>false</bool>
       </property>
       <property name="digitCount">
        <number>1</number>
       </property>
       <property name="segmentStyle">
        <enum>QLCDNumber::Outline</enum>
       </property>
       <property name="value" stdset="0">
        <double>0.000000000000000</double>
       </property>
       <property name="intValue" stdset="0">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>643</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MouseTrackerWindow</class>
   <extends>QWidget</extends>
   <header location="global">mousetrackerwindow.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LineWidthEdit</class>
   <extends>QWidget</extends>
   <header location="global">linewidthedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
